ROADMAP.md

```diff
@@ -1,30 +1,32 @@
-| Provide Stability | **[kapp-controller]** [Surface namespaces for deployed resources](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/535) | Early August |
-| Provide Stability | **[kapp-controller]** Version safeguards: To provide safeguards and visibility for kapp-controller and kubernetes versions. | Early August |
-| Package Author Improvements | **[kctrl]** kctrl commands for package authors. | Aug 2022 |
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) GA: Configuration authors can specify the valid range or format of the data values. | Late August |
-| Provide Stability | **[kapp-controller]** Minimize registry communications: To provide a more performant experience for users leveraging large PackageRepos. | Late August |
-
+`Last Updated: Aug 2022`
+|Theme|Feature|Stage|Timeline|
+|---|---|---|---|
+| Package Author Experience | **[kctrl]** [kctrl commands for package authors.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/632): CLI-based Package Author commands to enable  Package Authors easily create a Carvel package of their software | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[kctrl]** [kctrl dev deploy.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/2): Enable quick iteration on Package CRs and App CRs using kapp-controller CLI for local development | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents): Configuration authors can specify the valid range or format of the data values. || Q3 (Jul - Sep) |
+| Package Author Experience | **[carvel]** Carvel supports the ability to sign and verify assets (such as images, bundles, pkg/pkgr). |Awaiting Proposal| TBD |
+| Easy to Get Started | **[ytt]** [Guides & Examples](https://github.com/vmware-tanzu/carvel-ytt/issues/314): Provide more guides and examples so that ytt is easy to get started with and details how it can be incorporate in different workflows. [Epic](https://app.zenhub.com/workspaces/carvel-backlog-6013063a24147d0011410709/board?epics=173207060_314&filterLogic=any&repos=173207060) |Awaiting Proposal| TBD |
+| Stability | **[kapp]** [Do not exit on first error](https://github.com/vmware-tanzu/carvel-kapp/issues/426): Summarize all errors found at the end of execution |Build| Q3 (Jul-Sep) 2022|
+| Stability | **[kapp]** [Versioned resource based on a predetermined interval](https://github.com/vmware-tanzu/carvel-kapp/issues/224): kapp deploy to trigger a new versioned asset based on a predetermined interval | Build | Q3 (Jul-Sep) 2022|
+| Stability | **[kapp]** [Use kapp as Go module](https://github.com/vmware-tanzu/carvel-kapp/issues/564): Use as a Go module, help improve the error handling. | Awaiting Proposal | TBD |
```

Comment:
once this PR lands, I have a couple of other items that can be added to the roadmap. I think it'd just complicate things to try to do it now.